worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/clipboard/io/FastSchematicReaderV3.java

```diff
@@ -136,6 +139,27 @@ public Clipboard read(final UUID uuid, final Function<BlockVector3, Clipboard> c
                     this.dataVersion = this.dataInputStream.readInt();
                     this.dataFixer = ReaderUtil.getVersionedDataFixer(this.dataVersion, platform, platform.getDataVersion());
                 }
+                case "Metadata" -> {
+                    LinCompoundTag metadataCompoundTag =
+                            (LinCompoundTag) this.nbtInputStream.readTagPayload(NBTConstants.TYPE_COMPOUND, 2).toLinTag();
+
+                    LinCompoundTag worldEditTag = metadataCompoundTag.findTag("WorldEdit", LinTagType.compoundTag());
+                    if (worldEditTag == null) {
```

Comment:
Schematic's don't need to contain a WorldEdit compound in the metadata. Everything under metadata should be optional.

Reply:
Implemented ability for metadata sub-tags to be optional